docker-for-windows/index.md

```diff
@@ -13,12 +13,13 @@ redirect_from:
 title: Get started with Docker for Windows
 ---
 
-Welcome to Docker Desktop for Windows!
+Welcome to Docker Desktop!
 
-Docker is a full development platform for creating containerized apps, and
-Docker Desktop for Windows is the best way to get started with Docker _on Windows_.
+This page contains information about Docker Desktop for Windows Community (Stable and Edge) releases. For information about Docker Desktop Enterprise (DDE) releases, see [Docker Desktop Enterprise](/ee/desktop/).
```

Comment:
Should this be "Docker Desktop Community (Stable and Edge) releases?"

Reply:
Fixed in commit 59aa7e5.